src/query/expression/tests/it/kernel.rs

```diff
@@ -261,14 +261,18 @@ pub fn test_take_and_filter_and_concat() -> common_exception::Result<()> {
     let mut blocks = Vec::with_capacity(data_types.len());
     let mut filtered_blocks = Vec::with_capacity(data_types.len());
     for i in 0..num_blocks {
-        let len = rng.gen_range(5..100);
-        let filter = Column::random(&DataType::Boolean, len)
+        let len = rng.gen_range(2..100);
```

Comment:
Make sure we can coverage all the column types which introduced in https://github.com/datafuselabs/databend/blob/2eb820df9d8bfbbd4d9d767e72ca43b8f4b55f39/src/query/expression/src/kernels/concat.rs#L113-L331

We can change this `vec` to `match`? Then if someone add a new type will be test here:
```
let data_types = vec![
        DataType::Null,
        DataType::EmptyArray,
        DataType::EmptyMap,
        DataType::Boolean,
        DataType::String,
        DataType::Bitmap,
        DataType::Variant,
        DataType::Timestamp,
        DataType::Date,
        DataType::Number(NumberDataType::UInt8),
        DataType::Number(NumberDataType::UInt16),
        DataType::Number(NumberDataType::UInt32),
        DataType::Number(NumberDataType::UInt64),
        DataType::Number(NumberDataType::Int8),
        DataType::Number(NumberDataType::Int16),
        DataType::Number(NumberDataType::Int32),
        DataType::Number(NumberDataType::Int64),
        DataType::Number(NumberDataType::Float32),
        DataType::Number(NumberDataType::Float64),
        DataType::Decimal(DecimalDataType::Decimal128(DecimalSize {
            precision: 10,
            scale: 2,
        })),
        DataType::Decimal(DecimalDataType::Decimal128(DecimalSize {
            precision: 35,
            scale: 3,
        })),
        DataType::Nullable(Box::new(DataType::Number(NumberDataType::UInt32))),
        DataType::Nullable(Box::new(DataType::String)),
        DataType::Array(Box::new(DataType::Number(NumberDataType::UInt32))),
        DataType::Map(Box::new(DataType::Tuple(vec![
            DataType::Number(NumberDataType::UInt64),
            DataType::String,
        ]))),
    ];
```

Reply:
I'm adding a common function to cover all datatypes in other pr.

```
fn get_all_test_data_types() -> Vec<DataType> {
	...
}
```